docs/source/examples/shapes_images_positions.rst

```diff
@@ -1,5 +1,22 @@
 Shapes, Images and Positions
 =================================
+.. manim:: ShowScreenResolution
+    :save_last_frame:
+
+    class ShowScreenResolution(Scene):
+        def construct(self):
+            pyFrame = config["pixel_height"]  # 1080 default
+            pxFrame = config["pixel_width"]  # 1920 #default
```

Comment:
Nitpick (non-blocking): I found these variable names confusing: at first I thought they were both named pxFrame. At first I thought "px" stood for pixel and "py" stood for python. Also, these names do not technically adhere to PEP8 recommendations (snake_case is preferred for variable names over camelCase).

Reply:
ok, changed it